conftest.py

```diff
@@ -363,6 +363,17 @@ def zenodo_sandbox(client):
     )
 
 
+@pytest.fixture
+def dataverse_demo(client):
+    """Configure environment to use Dataverse demo environment."""
+    client.set_value(
+        'dataverse', 'access_token', '4ca13597-cf43-4815-8763-b64994058c19'
```

Comment:
Access token should probably be extracted into env variable and encrypted for the [CI](https://docs.travis-ci.com/user/encrypting-files/).

Reply:
This a free account on their demo server (same as the one above for Zenodo). I'd prefer to keep it like this for easier execution of tests in our local machines.